executors/http/executor.go

```diff
@@ -119,3 +121,71 @@ func (Executor) Run(l *log.Entry, aliases venom.Aliases, step venom.TestStep) (v
 	r.StatusCode = resp.StatusCode
 	return dump.ToMap(r, dump.WithDefaultLowerCaseFormatter())
 }
+
+
+// getRequest returns the request correctly set for the current executor
+func (e Executor) getRequest() (*http.Request, error) {
+	path := fmt.Sprintf("%s%s", e.URL, e.Path)
+	method := e.Method
+	if method == "" {
+		method = "GET"
+	}
+	if e.Body != "" && e.MultipartForm != nil {
+		return nil, fmt.Errorf("Cannot use both 'body' and 'multipart_form'")
+	}
+	var body *bytes.Buffer
+	var writer *multipart.Writer
+	if e.Body != "" {
+		body = bytes.NewBuffer([]byte(e.Body))
+	} else if e.MultipartForm != nil {
+		form := e.MultipartForm.(map[interface{}]interface{})
+		body = &bytes.Buffer{}
+		writer = multipart.NewWriter(body)
+		for key_, value_ := range form {
+			key := key_.(string)
```

Comment:
may be you should try to cast e.MultipartForm to (map[string]string) directly to avoid cas of keys and values.
Or you must check `key, ok := key_.(string)`